src/components/plots/plot.ts

```diff
@@ -54,6 +56,8 @@ module Plottable {
       this.clipPathEnabled = true;
       this.classed("plot", true);
       this._key2PlotDatasetKey = d3.map();
+      this._attrToExtents = d3.map();
+      this._extentProvider = (scale: Scale<any, any>) => this._extentsForScale(scale);
```

Comment:
Oh I was referring that _extentProvider maybe should not be an instance variable but a function instead, cause now we have this _extentProvider instance variable running around that can be set or gotten